Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -819,6 +819,13 @@ def audit_revision_and_version_scheme
 
   def patch_problems(patch)
     case patch.url
+    when %r{https?://github\.com/.+/.+/(?:commit|pull)/[a-fA-F0-9]*.(?:patch|diff)}
+      unless patch.url =~ /\?full_index=\w+$/
```

Comment:
I'm being overly pedantic by matching any param, not just a literal `1`. I guess it could just restrict it to that, heh.